stdlib/area/position.lua

```diff
@@ -132,6 +132,14 @@ function Position.manhattan_distance(pos1, pos2)
     return math.abs(pos2.x - pos1.x) + math.abs(pos2.y - pos1.y)
 end
 
+--- Whether 2 positions are equal
+-- @param pos1 the first position
+-- @param pos2 the second position
+-- @return true if positions are equal
+function Position.equals(pos1, pos2)
+    return (Position.distance_squared(pos1,pos2) == 0)
```

Comment:
This is a really expensive way to test it. Maybe add a line to test if pos1 ~= nil and pos1 == pos2, then do the distance_squared test.